### PR TITLE
trivial: powerd: reformat code with contrib/reformat-code.py

### PR DESCRIPTION
**Pull request by djcampello**
cc @bleungatchromium @g2pop @binaryNewts

### CODE REVIEW

File: plugins/powerd/fu-plugin-powerd.c [hunk 1]
```diff
@@ -31,28 +31,21 @@ fu_plugin_destroy (FuPlugin *plugin)
 gboolean
 fu_plugin_startup (FuPlugin *plugin, GError **error)
 {
-	FuPluginData *data = fu_plugin_get_data (plugin);
+	FuPluginData     *data = fu_plugin_get_data (plugin);
```
Comment (hughsie) [inline plugins/powerd/fu-plugin-powerd.c:34]: is that right? I thought we only block aligned in structs?

File: plugins/powerd/fu-plugin-powerd.c [hunk 2]
```diff
@@ -31,28 +31,21 @@ fu_plugin_destroy (FuPlugin *plugin)
...
-						     G_DBUS_PROXY_FLAGS_NONE,
-						     NULL,
-						     "org.chromium.PowerManager",
-						     "/org/chromium/PowerManager",
-						     "org.chromium.PowerManager",
-						     NULL,
-						     error);
+	data->proxy = g_dbus_proxy_new_for_bus_sync (
+	    G_BUS_TYPE_SYSTEM, G_DBUS_PROXY_FLAGS_NONE, NULL, "org.chromium.PowerManager",
+	    "/org/chromium/PowerManager", "org.chromium.PowerManager", NULL, error);
```
Comment (hughsie) [inline plugins/powerd/fu-plugin-powerd.c:40]: not sure i agree that is clearer....

File: plugins/powerd/fu-plugin-powerd.c (hunk 1 above)
Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c:34]: AlignConsecutiveDeclarations is global in the sense that affects structs and blocks of code the same

Comment (hughsie) [reply plugins/powerd/fu-plugin-powerd.c:34]: There's no way of making it non-global?

Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c:34]: not that I can find on https://clang.llvm.org/docs/ClangFormatStyleOptions.html :(

File: plugins/powerd/fu-plugin-powerd.c (hunk 2 above)
Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c]: done

File: plugins/powerd/fu-plugin-powerd.c (hunk 1 above)
Comment (hughsie) [reply plugins/powerd/fu-plugin-powerd.c:34]: maybe AlignConsecutiveDeclarations=false makes more sense then perhaps?

Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c]: done

File: plugins/powerd/fu-plugin-powerd.c [hunk 3]
```diff
@@ -61,10 +62,7 @@ fu_plugin_startup (FuPlugin *plugin, GError **error)
 }
 
 gboolean
-fu_plugin_update_prepare (FuPlugin *plugin,
-			  FwupdInstallFlags flags,
-			  FuDevice *device,
-			  GError **error)
+fu_plugin_update_prepare (FuPlugin *plugin, FwupdInstallFlags flags, FuDevice *device, GError **error)
```
Comment (hughsie) [inline plugins/powerd/fu-plugin-powerd.c:65]: that's 102 chars wide -- didn't we standardize on 80?

Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c:65]: This was left for avoiding huge changes to existent code... The piece responsible of this is the `PenaltyExcessCharacter: '1'` I'll remove now to go back to under 80

Comment (djcampello) [reply plugins/powerd/fu-plugin-powerd.c]: done

**Review by hughsie — APPROVED** [review]

**Pull request merged**